Wire-iOS/Sources/Helpers/UIBarButtonItem+StyleKit.swift

```diff
@@ -45,4 +45,57 @@ extension UIBarButtonItem {
         return item
     }
 
+    static func createHeaderDoneItem(
+        systemImage: Bool,
+        target buttonTarget: Any?,
+        action buttonAction: Selector?,
+        font buttonFont: FontSpec = FontSpec.headerRegularFont) -> UIBarButtonItem {
+
+            var rightBarButtonItem: UIBarButtonItem
+            if systemImage {
+                rightBarButtonItem = UIBarButtonItem(
+                    barButtonSystemItem: .done,
+                    target: buttonTarget,
+                    action: buttonAction)
+            } else {
+                rightBarButtonItem = UIBarButtonItem(
+                    title: "general.done".localized,
+                    style: .plain,
+                    target: buttonTarget,
+                    action: buttonAction)
+            }
+
+            let buttonStates: [UIControl.State] = [.normal, .highlighted, .disabled, .selected, .focused, .application, .reserved]
+
+            if let buttonFont = buttonFont.font {
+                buttonStates.forEach { buttonState in
+                    rightBarButtonItem.setTitleTextAttributes(
+                        [NSAttributedString.Key.font: buttonFont],
+                        for: buttonState)
+                }
+            }
+            return rightBarButtonItem
+    }
+
+    static func createHeaderEditItem(
```

Comment:
I would rename that to 
```suggestion
    static func createNavigationBarEditItem(
```

Reply:
Done.